src/FSharp.Formatting.Common/Menu.fs

```diff
@@ -0,0 +1,56 @@
+module FSharp.Formatting.Common.Menu
+
+open System
+open System.IO
+open FSharp.Formatting.Templating
+
+type MenuItem = { Link: string; Content: string }
+
+let private snakeCase (v: string) =
+    let replace = v.Replace(" ", "")
+
+    System
```

Comment:
I didn't catch this at first but spaces are not replaced by this code.
Leads to identifiers with spaces if the input has any.
Add `.Replace(" ", "")` before `.ToLower()`.